app/configurator/components/chart-controls/color-picker.tsx

```diff
@@ -75,45 +67,22 @@ const useColorPickerStyles = makeStyles((theme: Theme) => ({
     gap: 2,
     marginBottom: 2,
   },
-  input: {
-    color: theme.palette.grey[700],
-    borderColor: theme.palette.divider,
-    backgroundColor: theme.palette.grey[100],
-    fontSize: "0.875rem",
-    "&:focus": { outline: "none", borderColor: theme.palette.primary.main },
-  },
 }));
 
 export const ColorPicker = ({ selectedColor, colors, onChange }: Props) => {
-  const [inputColorValue, setInputColorValue] = useState(selectedColor);
-
+  const classes = useColorPickerStyles();
   const selectColor = useCallback(
     (_color) => {
-      setInputColorValue(_color);
       // Make sure onChange is only called with valid colors
```

Comment:
Is there no validation errors incase this fails?

Reply:
Before this PR, we had a text input to specify our own color – if it was invalid, it simply didn't update the state. Here we removed the text input completely to only allow the color from the active color palette, selectable via color swatch buttons, so they will always be correct 👀

Reply:
okay, so then by not just call the `d3Color()` directly instead of creating the callback?  doesnt seem like it does anything anymore 🤷‍♂️

Reply:
Good point!